CA_DataUploaderLib/ServerUploader.cs

```diff
@@ -37,6 +37,8 @@ public sealed class ServerUploader : ISubsystemWithVectorData
         private readonly IIOconf _ioconf;
         private readonly CommandHandler _cmd;
         private readonly TaskCompletionSource<PlotConnection> _connectionEstablishedSource = new();
+        private readonly Stopwatch _timeSinceLastLowFrequencyEvent = new();

```

Comment:
This watch should probably be specific to the special double values detection.

Reply:
done